src/DateTimePicker.jsx

```diff
@@ -132,6 +132,7 @@ class DateTimePicker extends React.Component {
 	*/
 	setDateTime(value) {
 		this.setState({ datetime: new Date(value) });
+		this.props.onChangeCallback(this.state.datetime, this.props.name);
```

Comment:
(1) I'll move this outta `setDateTime` and call it only when datetime state are being changed.
(2) `name` should probably removed, then -- that looks like a bit of product creep. Will take it out!
(3) Will look into adding the `event` arg just to start using the most common pattern.